src/Geometry.cpp

```diff
@@ -163,3 +250,19 @@ void Geometry::Destroy() const {
     glDeleteBuffers( 1, &colourBuffer );
     glDeleteBuffers( 1, &normalBuffer );
 }
+
+void Geometry::GetCorkTriMesh( CorkTriMesh& out ) {
+    // Dynamically create copies of geometry data
+    std::vector<GLuint>* nfaces = new std::vector<GLuint>( faces );
```

Comment:
do we need to delete these later

Reply:
Technically yeah but this is temporary and will be abstracted away later. This is just so it's convenient to test with for now